tox.ini

```diff
@@ -40,50 +40,43 @@ deps =
     devdeps: matplotlib>=0.0.dev0
     devdeps: numpy>=0.0.dev0
     devdeps: pandas>=0.0.dev0
-    # Oldest deps we pin against
-    oldestdeps: asdf-astropy<0.1.2
-    oldestdeps: asdf<2.9.0
-    oldestdeps: astropy<5.1.0
-    oldestdeps: beautifulsoup4<4.9.0
-    oldestdeps: cdflib<0.3.20
-    oldestdeps: dask[array]<2021.5.0
+    # oldestdeps we pin against to ensure correct versions
+    oldestdeps: asdf-astropy<0.3.0
+    oldestdeps: asdf<2.12.0
+    oldestdeps: astropy<5.3.0
+    oldestdeps: beautifulsoup4<4.12.0
+    oldestdeps: cdflib<0.4.5
+    oldestdeps: dask[array]<2022.6.0
     oldestdeps: drms<0.7
     oldestdeps: glymur<0.9.2
-    oldestdeps: h5netcdf<0.12
+    oldestdeps: h5netcdf<1.1.0
     oldestdeps: matplotlib<3.6.0
     oldestdeps: numpy<1.22.0
-    oldestdeps: opencv-python<4.8.0.72
-    oldestdeps: pandas<1.3.0; platform_system!='Darwin' or platform_machine!='arm64'
-    # pandas 1.4.0 is the first with wheels for macOS arm64
-    oldestdeps: pandas<1.5.0; platform_system=='Darwin' and platform_machine=='arm64'
+    oldestdeps: opencv-python<4.7.0.68
+    oldestdeps: pandas<1.5.0
     oldestdeps: parfive<1.3.0
     oldestdeps: pytest-xdist<2.1
-    oldestdeps: pytest<7.0
+    oldestdeps: pytest<7.2
     oldestdeps: python-dateutil<2.9.0
-    oldestdeps: scikit-image<0.19.0; platform_system!='Darwin' or platform_machine!='arm64'
+    oldestdeps: scikit-image<0.20.0; platform_system!='Darwin' or platform_machine!='arm64'
     # scikit-image 0.20 is the first with wheels for macOS arm64
     oldestdeps: scikit-image<0.21.0; platform_system=='Darwin' and platform_machine=='arm64'
-    oldestdeps: scipy<1.8.0
-    oldestdeps: tqdm<4.33.0
-    oldestdeps: zeep<3.5.0
-    # These are specific extras we use to run in specific factors
-    online: pytest-rerunfailures
-    online: pytest-timeout
-    online: matplotlib>=3.8
+    oldestdeps: scipy<1.9.0
+    oldestdeps: tqdm<4.65.0
+    oldestdeps: zeep<4.2.0
     # Figure tests need a tightly controlled environment
     # These are pinned to the minimum version that we support
-    figure-!devdeps: astropy==5.1.1
+    figure-!devdeps: astropy==5.3.4
     figure-!devdeps: matplotlib==3.5.3
```

Comment:
Should we bump Matplotlib version too? (even if it's only used in the tests)

Reply:
I can do so.